interface/VideoCommonDefs.h

```diff
@@ -25,6 +25,10 @@
 
 #include <stdint.h>
 
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
```

Comment:
But there need to use the macro, '__ENABLE_CAPI__'.
I am not sure that that is available.